addons/website_slides/models/gamification_karma_tracking.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class KarmaTracking(models.Model):
+    _inherit = 'gamification.karma.tracking'
+
+    def _get_origin_selection_values(self):
+        return (
+            super(KarmaTracking, self)._get_origin_selection_values()
+            + [('slide.slide', 'Quiz'), ('slide.channel', self.env['slide.channel']._description)]
```

Comment:
"Quiz" alone could be a bit confusing as we may have quiz in multiple modules, maybe "E-learning Quiz" (or "Course Quiz" ?) ?